### PR TITLE
Remove motsu from the repo and just import from crates.io

### LINKED ISSUE #455: Move motsu to its own repo
### What is the feature you would like to see?

It's easier for external contributions and keeping things more organized if we move motsu to a new `stylus-test-helpers` repository and publish the crate from there.
We might eventually do the same for `e2e` if there's enough demand.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

### PR DESCRIPTION
**Pull request by ggonzalez94**
Resolves #455 

Next Steps
- [x] Make the new repo public (https://github.com/OpenZeppelin/stylus-test-helpers)
- [x] Publish `0.2.1` for motsu and motsu-proc from the new repo?
- [ ] Merge this PR

### CODE REVIEW

**Review by 0xNeshi — APPROVED** [review]
The broken link should be fixed once the new repo is created.
Other than that, lgtm!

File: .github/workflows/check-publish.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ permissions:
   contents: read
 on:
   push:
-    branches: [ main, release/* ]
+    branches: [main, release/*]
```
Comment (qalisander) [inline .github/workflows/check-publish.yml:7]: I see we have different formattings for `*.yml`.
I know that @0xNeshi also prefers to format like it was before

Comment (0xNeshi) [reply .github/workflows/check-publish.yml:7]: I'm fine either way!
Let me know which formatter you're using for _.yml_ files, and I'll install it locally

Comment (ggonzalez94) [reply .github/workflows/check-publish.yml:7]: same here!

Comment (ggonzalez94) [reply .github/workflows/check-publish.yml:7]: I know this was discussed in the past. But I think we should actually commit our `.pre-commit-config.yaml` to make things more consistent for everyone

**Review by qalisander — APPROVED** [review]
LGTM!

**Pull request merged**